javaagent-tooling/javaagent-tooling-java9/src/main/java/io/opentelemetry/javaagent/tooling/ModuleOpener.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.tooling;
+
+import static java.util.logging.Level.FINE;
+import static java.util.logging.Level.WARNING;
+
+import java.lang.instrument.Instrumentation;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.logging.Logger;
+
+public class ModuleOpener {
```

Comment:
Shouldn't there also be a no-op implementation of this class in the non-java9 `tooling` project?
Otherwise we'll get a `ClassNotFoundException` on Java 8 I think?

This is likely not covered by CI tests yet because we don't rund the indy tests on multiple java versions.

Reply:
The other classes from this module are used with a try/catch(UnsupportedClassVersionError), but here we can also use `net.bytebuddy.utility.JavaModule#isSupported`, I've added that in 9c251f0464

Reply:
Interesting, I though that the `tooling-java9` project would be bundled as [Multi Release Jar](https://openjdk.org/jeps/238)